src/main/java/com/softserve/kickscootertrip/model/dto/TripDto.java

```diff
@@ -0,0 +1,21 @@
+package com.softserve.kickscootertrip.model.dto;
+
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+public class TripDto {
+    private UUID tripId;
+    private UUID userId;
+    private UUID scooterId;
+    private LocalDateTime start;
+    private LocalDateTime finish;
+    private long tripTime;
+    private double distance;
+    private int tripsNumber;
```

Comment:
what is the purpose of this field?

Reply:
Maybe Sales microservice will give some discounts for every tens ride